lib/screens/v2/candidate_generator/dup/alerts.ex

```diff
@@ -53,18 +55,25 @@ defmodule Screens.V2.CandidateGenerator.Dup.Alerts do
            fetch_routes_by_stop_fn.(stop_id, now, route_type_filter),
          subway_route_ids_at_stop =
            for(%{route_id: id} <- subway_routes_at_stop, id != "Mattapan", do: id),
-         {:ok, alerts} <- fetch_alerts_fn.(route_ids: subway_route_ids_at_stop),
+         {:ok, alerts} <- fetch([route_ids: subway_route_ids_at_stop], fetch_alerts_fn),
          {:ok, stop_sequences} <-
            fetch_parent_station_sequences_fn.(stop_id, subway_route_ids_at_stop) do
       alerts
-      |> Enum.filter(&relevant_alert?(&1, config, stop_sequences, subway_routes_at_stop, now))
+      |> relevant_alerts(config,
+        stop_sequences: stop_sequences,
+        subway_routes_at_stop: subway_routes_at_stop,
+        now: now
+      )
```

Comment:
Not sure I follow. Do you mean just getting rid of `stop_sequences:` from the parameter list? That won't work because it is part of the `opt` list. The only parameters that are required are `alerts` and `config` because those are the only common parameters in all widgets. Any after that are just any values needed by that particular widget and need to be given as a keyword list.

Reply:
To elaborate, this is a sometimes-confusing feature / syntactic sugar in elixir:
```ex
relevant_alerts(alerts, config, stop_sequences: stop_sequences, subway_routes_at_stop: subway_routes_at_stop, now: now)
```
is exactly equivalent to
```ex
relevant_alerts(alerts, config, [stop_sequences: stop_sequences, subway_routes_at_stop: subway_routes_at_stop, now: now])
```
That is, all of the "named" arguments are actually part of one keyword-list argument, usually named `opts`.

[relevant docs](https://elixir-lang.org/getting-started/keywords-and-maps.html#keyword-lists), see the "Furthermore, when a keyword list is the last argument of a function..." part